docs/sequence/error_scenarios/PISPLinkingRegisterCredentialPhase.puml

```diff
@@ -0,0 +1,297 @@
+@startuml
+
```

Comment:
cut off everything above Authentication-53

Reply:
I prefer having all the prerequsite happy flow up til the error for viewing insolation.